.castor/infra.php

```diff
@@ -0,0 +1,224 @@
+function generate_certificates(
+    #[AsOption(description: 'Force the certificates re-generation without confirmation', shortcut: 'f')]
+    bool $force = false,
+): void {
+    if (file_exists(variable('root_dir') . '/infrastructure/docker/services/router/etc/ssl/certs/cert.pem') && !$force) {
+        io()->comment('SSL certificates already exists.');
+        io()->note('Run "castor infra:generate-certificates --force" to generate new certificates.');
+
+        return;
+    }
+
+    if ($force) {
+        if (file_exists($f = variable('root_dir') . '/infrastructure/docker/services/router/etc/ssl/certs/cert.pem')) {
+            io()->comment('Removing existing certificates in infrastructure/docker/services/router/etc/ssl/certs/*.pem.');
+            unlink($f);
+        }
+
+        if (file_exists($f = variable('root_dir') . '/infrastructure/docker/services/router/etc/ssl/certs/key.pem')) {
+            unlink($f);
+        }
+    }
+
+    $finder = new ExecutableFinder();
+    $mkcert = $finder->find('mkcert');
+
+    if ($mkcert) {
+        $pathCaRoot = capture(['mkcert', '-CAROOT']);
+
+        if (!is_dir($pathCaRoot)) {
+            io()->warning('You must have mkcert CA Root installed on your host with "mkcert -install" command.');
+
+            return;
+        }
+
+        $rootDomain = variable('root_domain');
+
+        run([
+            'mkcert',
+            '-cert-file', 'infrastructure/docker/services/router/etc/ssl/certs/cert.pem',
+            '-key-file', 'infrastructure/docker/services/router/etc/ssl/certs/key.pem',
+            $rootDomain,
+            "*.{$rootDomain}",
+            ...variable('extra_domains'),
+        ]);
+
+        io()->success('Successfully generated SSL certificates with mkcert.');
+
+        if ($force) {
+            io()->note('Please restart the infrastructure to use the new certificates with "castor up" or "castor start".');
+        }
+
+        return;
+    }
+
+    run(['infrastructure/docker/services/router/generate-ssl.sh']);
+
+    io()->success('Successfully generated self-signed SSL certificates in infrastructure/docker/services/router/etc/ssl/certs/*.pem.');
+    io()->comment('Consider installing mkcert to generate locally trusted SSL certificates and run "castor infra:generate-certificates --force".');
+
+    if ($force) {
+        io()->note('Please restart the infrastructure to use the new certificates with "castor up" or "castor start".');
+    }
+}
+
+#[AsTask(description: 'Starts the workers', namespace: 'infra:worker', name: 'start')]
+function workers_start(): void
+{
+    $workers = get_workers();
+
+    if (!$workers) {
+        return;
+    }
+
+    run([
+        'docker',
+        'update',
+        '--restart=unless-stopped',
+        ...$workers,
+    ], quiet: true);
+
+    run([
+        'docker',
+        'start',
+        ...$workers,
+    ], quiet: true);
+}
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏There are no worker on this project 😬

Reply:
You really want to remove them from the infrastructure? That would complexify future updates of the stack without real gain IMO

Reply:
I agree with @damienalexandre here. If we don't use it, it does not worth keeping it